product_equivalent_category/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_equivalent_category_manager,product.equivalent.category.manager,model_product_equivalent_category,base.group_partner_manager,1,1,1,1
+access_product_equivalent_category_user,product.equivalent.category.user,model_product_equivalent_category,,1,0,0,0
```

Comment:
Add an extra line to fix Travis warning
************* Module product_equivalent_category
product_equivalent_category/security/ir.model.access.csv:1: [W7908(missing-newline-extrafiles), ]  Missing newline